### PR TITLE
Ignore searchDomain for localhost and IP

### PR DESCRIPTION
**Pull request by jkrems**
Backport of 8cfb230

### CODE REVIEW

File: src/helpers.coffee [hunk 1]
```diff
@@ -71,6 +71,18 @@ cleanProperty = (obj, value, key) ->
   delete defaults.endpointDefaults
   { defaults, endpointDefaults }
 
+IPv4 = /^\d{1,3}.\d{1,3}.\d{1,3}.\d{1,3}$/
```
Comment (saumitrab) [inline src/helpers.coffee]: escape the dots?

Comment (jkrems) [reply src/helpers.coffee]: Heh, nice catch. Might do it in the master branch.

Comment (jkrems) [reply src/helpers.coffee]: For context: I considering going as far as only testing if the hostname starts with a digit so I'm not *too* concerned that the pattern might match unindented strings.

Comment (saumitrab) [reply src/helpers.coffee]: 👍

**Pull request merged**